texttospeech/synthesize_text.rb

```diff
@@ -79,12 +79,16 @@ def synthesize_ssml ssml:
   elsif command == "ssml"
     synthesize_text ssml: text
   else
-    puts <<-usage
-Usage: ruby synthesize_text.rb (text TEXT | ssml SSML)
+    puts <<~USAGE
+      Usage: ruby synthesize_text.rb (text TEXT | ssml SSML)
 
-Example:
-  ruby synthesize_text.rb text "hello"
-  ruby synthesize_text.rb ssml "<speak>Hello there.</speak>"
-    usage
+      Example:
+        ruby synthesize_text.rb text "hello"
+        ruby synthesize_text.rb ssml "<speak>Hello there.</speak>"
+    USAGE
   end
 end
+
+
+"asdfasdfasdfasdfasdfasdfasdfasdfasdfasdasdfasdfasdfasdfasdfasdf" +
+  "asdfasdfasdfasdfasdfasdfasdfasdfasdfasdasdfasdfasdfasdfasdfasdf"
```

Comment:
what's this for?

Reply:
Lol, trying to find the cop that determines multi-line string concatenation